### PR TITLE
add some documentation sections, fix a minor CSS issue

### PR DESCRIPTION
**Pull request by probablyup**

### CODE REVIEW

**Comment by probablyup** [comment]
Hmm, how do I update the integration test base screenshots since I added some sections?

**Comment by mxstbr** [comment]
`yarn test integration -u` I think? We should document that in the CONTRIBUTING.md, wanna submit a follow-up PR once you've figured it out? :angel:

**Comment by probablyup** [comment]
@mxstbr worked perfectly, thanks!

File: sections/basics/components-as-selectors.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import md from 'components/md'
+
+const ComponentSelectors = () => md`
```
Comment (mxstbr) [inline sections/basics/components-as-selectors.js]: Don't we have docs about this somewhere already? I think in the FAQ?

The FAQ also has a better example than the one here, which shows how to target oneself which imo makes a lot more sense with styled-components than targeting other components.

Comment (probablyup) [reply sections/basics/components-as-selectors.js]: Oh yeah, you're right. I would not have thought to look for it in the FAQ... should we move the existing one to a more traffic'ed section?

Comment (mxstbr) [reply sections/basics/components-as-selectors.js]: Yeah maybe, let's move it to the advanced section and use the FAQ entry there instead? I don't really feel like this is a basic thing imo.

Comment (philpl) [reply sections/basics/components-as-selectors.js]: Agreed :) it it’d fit in better in “Advanced”

File: README.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ npm run dev
 
 > Note: This requires Node.js and npm to be set up locally, see [nodejs.org](https://nodejs.org) for more information.
 
+### Updating the visual diffs
+
+If you add a new section or materially change the website, it probably will trigger the image comparison diff snapshot to fail. These can be updated via:
+
+```sh
+yarn interaction -u
```
Comment (gribnoysup) [inline README.md]: I think it should be `integration` 😸

**Comment by probablyup** [comment]
@mxstbr updated per PR comments

**Pull request merged**